packages/strapi-generate-plugin/files/admin/src/containers/Initializer/tests/index.test.js

```diff
@@ -0,0 +1,25 @@
+// import React from 'react';
+// import { mount, shallow } from 'enzyme';
+// import pluginId from '../../pluginId';
+
+// import Initializer from '../index';
+
```

Comment:
Is this test enough ? Or can't this file be deleted ?

Reply:
Well, this file is generated when you create a new plugin so it can't be deleted, and it's commented in purpose